coralproject/talk
# [CORL-2833]: Update allowed origins input to Textarea

## What does this PR do?

These changes update the allowed origins input for sites in the admin to a text area to improve readability when creating and editing the allowed origins for a site.

## These changes will impact:


## What changes to the GraphQL/Database Schema does this PR introduce?

none

## Does this PR introduce any new environment variables or feature flags?

no

## If any indexes were added, were they added to `INDEXES.md`?

n/a

## How do I test this PR?

You can add a new site and see that there is a text area for entering allowed origins under `Site permitted domains`. See that the site creates as expected. Validation errors should show as expected for invalid domains, etc.

Edit a site and add/remove allowed origins. Save and see that it updates as expected.

Add a lot of domains and see that it is still readable.

## Where any tests migrated to React Testing Library?

no

## How do we deploy this PR?
